commands/http/handler.go

```diff
@@ -153,7 +175,7 @@ func guessMimeType(res cmds.Response) (string, error) {
 	return mimeTypes[enc], nil
 }
 
-func sendResponse(w http.ResponseWriter, req cmds.Request, res cmds.Response) {
+func sendResponse(w http.ResponseWriter, r *http.Request, req cmds.Request, res cmds.Response) {
```

Comment:
the ordering here feels weird. maybe switch res and req to stay consistent? http{resp, req}, cmds{resp, req}